CoCSharp.Client/RemoteServer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Net;
+using System.Net.Sockets;
+
+namespace CoCSharp.Client
+{
+    class RemoteServer
```

Comment:
Yeah it was in your commit, which was then added to the branch, which meant that we had some random useless class in the source.